packages/spark/generic/_normalize.scss

```diff
@@ -0,0 +1,465 @@
+/*! normalize.css v5.0.0 | MIT License | github.com/necolas/normalize.css */
```

Comment:
As per our conversation, this normalize file *could* be brought in as a package.
There are no custom styles in this unlike our `reset` file

hasn't been a problem so far, so this is fine, it's just something to think about in the future